_data/team/roles/outreach.yml

```
@@ -0,0 +1,42 @@
+role: Outreach
+subroles:
+  - subrole: Workshops and meetings
+    tasks:
+    - Identifying workshop and event opportunities
+    - Organising workshops and events
+    - Gathering and coordinating volunteers
+    - Managing content, presentations and teaching at workshops
+    current_leads:
+      - Jenna Swarthout Goddard
+    current_members:
+      - Micaela Matta
+      - All core devs
+    historical_members:
+      - Irfan Alibay
+      - Micaela Matta
+    
+  
+  - subrole: Mentoring
```

I would rename this sub role to
```suggestion
  - subrole: Mentoring programs
```
because it is about the organizational aspects (eg being a GSoC Org admin). Otherwise it sounds as if the actual mentoring is included. However, that's something that many coredevs do.